docs/modules/ROOT/pages/hazelcast-platform-operator-external-backup-restore.adoc

```diff
@@ -0,0 +1,500 @@
+= External Backup and Restore with Hazelcast
+:page-layout: tutorial
+:page-product: operator
+:page-categories: Cloud Native
+:page-lang: go, java, node, python
+:page-enterprise: true
+:page-est-time: 20 mins
+:description: Migrate Hazelcast backup data to/from blob storages that cloud providers maintain, within Hazelcast cluster running in Kubernetes.
+
+{description}
+
+== Context
+In this tutorial you will learn:
+
+- Deploying Hazelcast with persistence enabled.
+
+- Creating persistence enabled map in the Hazelcast.
+
+- Backing data up from the map into the external storage.
+
+- Restarting the Hazelcast cluster with the restored data from the external storage.
+
+There are two available options for Persistence feature of Hazelcast Platform Operator:
+
+- *Local* - backup the data to/from PVC.
+
+- *External* - backup the data to/from external(blob) storages.
+
+You'll see the *External* option of the Persistence feature.
+
+== Before you Begin
+
+Before starting this tutorial, make sure that you meet the following prerequisites:
+
+* Up and running https://kubernetes.io/[Kubernetes] cluster
+* Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
+* Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]
+* Created blob storage and access credentials in one of the cloud providers: https://aws.amazon.com/s3/[AWS] - https://cloud.google.com/storage/[GCP] - https://azure.microsoft.com/en-us/services/storage/blobs/[Azure]
+
+== Step 1. Start the Hazelcast Cluster
+
+=== Create License Secret
```

Comment:
Throughout this tutorial, we can remove the third-level headings and turn them into numbered steps.